solidity/contracts/system/TBTCSystem.sol

```diff
@@ -79,19 +81,21 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
     uint64[] lotSizesSatoshis = [10**5, 10**6, 10**7, 2 * 10**7, 5 * 10**7, 10**8]; // [0.001, 0.01, 0.1, 0.2, 0.5, 1.0] BTC
 
     uint256 constant governanceTimeDelay = 48 hours;
+    uint256 constant keepFactoriesUpgradeabilityPeriod = 180 days;
```

Comment:
Can't we have it as a public constant instead of exposing `getKeepFactoriesUpgradeabilityPeriod`?

Reply:
It's consistent with other getters, e.g. getGovernanceTimeDelay, getPriceFeedGovernanceTimeDelay.

Reply:
I can remove all getters and update constants to public, but I'm not sure if it was implemented like this on purpose.

Reply:
Alright, fair enough.